<a href="https://colab.research.google.com/github/Alissonfersoa/python-exercicios/blob/master/Exercicios_Iniciante_Python_Lista_2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

***Exercicios para iniciantes em Python***

Lista de Estrutura De Decisão

Todos os exercicios foram tirados do link abaixo:

https://wiki.python.org.br/EstruturaDeDecisao

Notebook desenvolvido por Alisson Fernando Soares Batista

1. Faça um Programa que peça dois números e imprima o maior deles.

In [ ]:
num1 = float(input('Digite um número: '))
num2 = float(input('Digite outro número: '))

#forma lista com os valores
lista_num = [num1, num2]

print('O maior valor é: ',max(lista_num))

Digite um número: 23.5
Digite outro número: 22
O maior valor é:  23.5


2. Faça um Programa que peça um valor e mostre na tela se o valor é positivo ou negativo.

In [ ]:
num = float(input('Digite um número: '))

if num>=0:
  print('Número digitado é positivo!')
else:
  print('Número digitado é negativo!')

Digite um número: -4
Número digitado é negativo!


3. Faça um Programa que verifique se uma letra digitada é "F" ou "M". Conforme a letra escrever: F - Feminino, M - Masculino, Sexo Inválido.

In [ ]:
letra = str(input('Digite uma letra: '))

if letra == 'F':
  print('F - Feminino')
elif letra =='M':
  print('M - Masculino')
else :
  print('Sexo Inválido!')


Digite uma letra: S
Sexo Inválido!


4. Faça um Programa que verifique se uma letra digitada é vogal ou consoante.

In [ ]:
letra = input('Digite um caractere: ')

if letra == 'a' or letra == 'e' or letra == 'i' or letra == 'o' or letra == 'u':
  print('O caractere ', letra, ' é uma vogal!')
else:
  print('O caractere ', letra, ' é uma consoante!')

Digite um caractere: D
O caractere  D  é uma consoante!


5. Faça um programa para a leitura de duas notas parciais de um aluno. O programa deve calcular a média alcançada por aluno e apresentar:

*   A mensagem "Aprovado", se a média alcançada for maior ou igual a sete;
*   A mensagem "Reprovado", se a média for menor do que sete;
*   A mensagem "Aprovado com Distinção", se a média for igual a dez.

In [ ]:
#usando função round para arredondar valores
nota1 = float(input('Digite a primeira nota: '))
nota2 = float(input('Digite a segunda nota: '))

notas = [nota1, nota2]

media = sum(notas)/len(notas)

if media >= 7:
  print('Aluno Aprovado! média: ', round(media,1))
elif media == 10:
  print('Aluno Aprovado com Distinção, média :', round(media,1))
else:
  print('Aluno reprovado! média: ', round(media,1))

Digite a primeira nota: 10
Digite a segunda nota: 8.9
Aluno Aprovado! média:  9.4


6. Faça um Programa que leia três números e mostre o maior deles.

In [ ]:
num1 = float(input('Digite um número: '))
num2 = float(input('Digite o segundo número: '))
num3 = float(input('Digite o terceiro número: '))

#usando comando lista para armazenar os valores em matriz
lista = [num1, num2, num3]

print('O maior número é: ',max(lista))


Digite um número: 4
Digite o segundo número: 5
Digite o terceiro número: 5.5
O maior número é:  5.5


7. Faça um Programa que leia três números e mostre o maior e o menor deles.

In [ ]:
num1 = float(input('Digite o primeiro número: '))
num2 = float(input('Digite o segundo número: '))
num3 = float(input('Digite o terceiro número: '))

lista = [num1, num2, num3]

print('O maior número é: ', max(lista))
print('O menor número é: ', min(lista))

Digite o primeiro número: 3
Digite o segundo número: 4
Digite o terceiro número: 7
O maior número é:  7.0
O menor número é:  3.0


8. Faça um programa que pergunte o preço de três produtos e informe qual produto você deve comprar, sabendo que a decisão é sempre pelo mais barato.

In [ ]:
p1 = float(input('Digite o 1° preço: '))
p2 = float(input('Digite o 2° preço: '))
p3 = float(input('Digite o 3° preço: '))


def menor():

  if p1 < p2 and p3 and p1:
    print('O produto 1 é o mais barato!!')
  elif p2 < p1 and p3:
    print('O produto 2 é o mais barato!!')
  elif p3 < p1 and p2:
    print('O produto 3 é o mais barato!!')
  elif p1 == p2 and p1 and p2 < p3:
    print('O produto 1 e 2 são os mais baratos!!')
  elif p1 == p3 and p1 and p3 < p2:
    print('O produto 1 e 3 são os mais baratos!!')
  elif p2 == p3 and p2 and p3 < p1:
    print('O produto 1 e 2 são os mais baratos!!')
  else:
    print('Todos os preços são iguais')
menor()

Digite o 1° preço: 12
Digite o 2° preço: 11
Digite o 3° preço: 23
O produto 2 é o mais barato!!


9. Faça um Programa que leia três números e mostre-os em ordem decrescente.

In [ ]:
a = float(input('Digite o primeiro número: '))
b = float(input('Digite o segundo número: '))
c = float(input('Digite o terceiro número: '))


if a < c:
    a, c = c, a
if a < b:
    a, b = b, a
if b < c:
    b, c = c, b

print('A ordem decrescente é: ',a,' | ',b,' | ',c)

Digite o primeiro número: 23
Digite o segundo número: 1
Digite o terceiro número: 0.66
A ordem decrescente é:  23.0  |  1.0  |  0.66


10. Faça um Programa que pergunte em que turno você estuda. Peça para digitar M-matutino ou V-Vespertino ou N- Noturno. Imprima a mensagem "Bom Dia!", "Boa Tarde!" ou "Boa Noite!" ou "Valor Inválido!", conforme o caso.


In [ ]:
turno = (input('Digite qual o turno que estuda (M - Matutino / V - Vespertino / N - Noturno): '))

def mensagem():
  if turno == 'm':
    print('Bom Dia!!')
  elif turno == 'v':
    print('Boa Tarde!!')
  elif turno == 'n':
    print('Boa Noite!!')
  else:
    print('Valor Inválido!!')
mensagem()

Digite qual o turno que estuda (M - Matutino / V - Vespertino / N - Noturno): m
Bom Dia!!


11. As Organizações Tabajara resolveram dar um aumento de salário aos seus colaboradores e lhe contraram para desenvolver o programa que calculará os reajustes.
Faça um programa que recebe o salário de um colaborador e o reajuste segundo o seguinte critério, baseado no salário atual:
salários até R$ 280,00 (incluindo) : 

* aumento de 20%
*   salários entre R 280,00 e R 700,00 : aumento de 15%
* salários entre R 700,00 e R 1500,00 : aumento de 10%
* salários de R 1500,00 em diante : aumento de 5% Após o aumento ser realizado, informe na tela:
* o salário antes do reajuste;
* o percentual de aumento aplicado;
* o valor do aumento;
* o novo salário, após o aumento.




In [ ]:
salario_atual = float(input('Digite o salário atual: '))

#condição para o valor do salario atual
if salario_atual <= 280.00:
  porcentagem = salario_atual * 0.20
  resultado = salario_atual + porcentagem
  print('====== Extrato reajuste ======')
  print('---- Seu salário antes do reajuste: R$', salario_atual)
  print('---- Percentual de aumento: ', 20, '%')
  print('---- O valor do aumento é: R$', porcentagem)
  print('---- Com o reajuste, seu salário é: R$',resultado)

elif salario_atual > 280.00 and salario_atual <= 700.00:
  porcentagem = salario_atual * 0.15
  resultado = salario_atual + porcentagem
  print('====== Extrato reajuste ======')
  print('---- Seu salário antes do reajuste: R$', salario_atual)
  print('---- Percentual de aumento: ', 15, '%')
  print('---- O valor do aumento é: R$', porcentagem)
  print('---- Com o reajuste, seu salário é: R$',resultado)

elif salario_atual > 700.00 and salario_atual <= 1500.00:
  porcentagem = salario_atual * 0.10
  resultado = salario_atual + porcentagem
  print('====== Extrato reajuste ======')
  print('---- Seu salário antes do reajuste: R$', salario_atual)
  print('---- Percentual de aumento: ', 10, '%')
  print('---- O valor do aumento é: R$', porcentagem)
  print('---- Com o reajuste, seu salário é: R$',resultado)

elif salario_atual > 1500.00:
  porcentagem = salario_atual * 0.05
  resultado = salario_atual + porcentagem
  print('====== Extrato reajuste ======')
  print('---- Seu salário antes do reajuste: R$', salario_atual)
  print('---- Percentual de aumento: ', 5, '%')
  print('---- O valor do aumento é: R$', porcentagem)
  print('---- Com o reajuste, seu salário é: R$',resultado)

Digite o salário atual: 1505
====== Extrato reajuste ======
---- Seu salário antes do reajuste: R$ 1505.0
---- Percentual de aumento:  5 %
---- O valor do aumento é: R$ 75.25
---- Com o reajuste, seu salário é: R$ 1580.25


12. Faça um programa para o cálculo de uma folha de pagamento, sabendo que os descontos são do Imposto de Renda, que depende do salário bruto (conforme tabela abaixo) e 3% para o Sindicato e que o FGTS corresponde a 11% do Salário Bruto, mas não é descontado (é a empresa que deposita). O Salário Líquido corresponde ao Salário Bruto menos os descontos. O programa deverá pedir ao usuário o valor da sua hora e a quantidade de horas trabalhadas no mês.
Desconto do IR:
* Salário Bruto até 900 (inclusive) - isento
* Salário Bruto até 1500 (inclusive) - desconto de 5%
* Salário Bruto até 2500 (inclusive) - desconto de 10%
* Salário Bruto acima de 2500 - desconto de 20% Imprima na tela as informações, dispostas conforme o exemplo abaixo. No exemplo o valor da hora é 5 e a quantidade de hora é 220.

            Salário Bruto: (5 * 220)        : R$ 1100,00
            (-) IR (5%)                     : R$   55,00  
            (-) INSS ( 10%)                 : R$  110,00
            FGTS (11%)                      : R$  121,00
            Total de descontos              : R$  165,00
            Salário Liquido                 : R$  935,00

In [ ]:
# ajustar código

#define variaveis
#considerando as horas como 220 e o salario hora de 5
salario_hora = float(input('Digite o valor da hora de trabalho: '))
horas = float(input('Digite a quantidade de hora trabalhada no mês: '))

salario_bruto = (salario_hora*horas)


def descontos(salario_bruto):
    descontoSindicato = (salario_bruto/100)*3
    descontoFGTS = (salario_bruto/100)*11
    descontoINSS = (salario_bruto/100)*10
    descontoIR = 0
    if salario_bruto <= 900:
        salario_liquido = salario_bruto - descontoSindicato - descontoINSS - descontoFGTS
        
    elif salario_bruto <= 1500:
        descontoIR = (salario_bruto/100)*5
        salario_liquido = salario_bruto - descontoSindicato - descontoIR - descontoINSS - descontoFGTS
        
    elif salario_bruto <= 2500:
        descontoIR = (salario_bruto/100)*10
        salario_liquido = salario_bruto - descontoSindicato - descontoIR - descontoINSS - descontoFGTS
        
    else:
        descontoIR = (salario_bruto/100)*20
        salario_liquido = salario_bruto - descontoSindicato - descontoIR - descontoINSS - descontoFGTS

    totaldesconto = (descontoSindicato + descontoIR + descontoINSS + descontoFGTS)
    imprime_desconto(salario_bruto, descontoIR, descontoSindicato, descontoINSS, descontoFGTS, totaldesconto, salario_liquido)

def imprime_desconto(salario_bruto, descontoIR, descontoSindicato, descontoINSS, descontoFGTS, totaldesconto, salario_liquido):
    print('''
    Salário Bruto: (5*220)          :  %.2f
    (-) IR (5%)                     :  %.2f
    (-) Sindicato                   :  %.2f
    (-) INSS ( 10%)                 :  %.2f
    FGTS (11%)                      :  %.2f
    Total de descontos              :  %.2f
    Salário Liquido                 :  %.2f

    '''%(salario_bruto, descontoIR, descontoSindicato, descontoINSS, descontoFGTS, totaldesconto, salario_liquido))

descontos(salario_bruto)

13. Faça um Programa que leia um número e exiba o dia correspondente da semana. (1-Domingo, 2- Segunda, etc.), se digitar outro valor deve aparecer valor inválido.

In [ ]:
#define variaveis
print('::: Define dia correspondente :::')
numsemana = int(input('Digite um número: '))

if numsemana == 1:
  print('1 - Domingo')
elif numsemana ==2:
  print('2 - Segunda')
elif numsemana ==3:
  print('3 - Terça')
elif numsemana ==4:
  print('4 - Quarta')
elif numsemana ==5:
  print('5 - Quinta')
elif numsemana ==6:
  print('6 - Sexta')
elif numsemana ==7:
  print('7 - Sabádo')
else:
  print('Válor Inválido!!')

::: Define dia correspondente :::
Digite um número: 1
1 - Domingo


14. Faça um programa que lê as duas notas parciais obtidas por um aluno numa disciplina ao longo de um semestre, e calcule a sua média. A atribuição de conceitos obedece à tabela abaixo:
  Média de Aproveitamento  Conceito
  Entre 9.0 e 10.0        A
  Entre 7.5 e 9.0         B
  Entre 6.0 e 7.5         C
  Entre 4.0 e 6.0         D
  Entre 4.0 e zero        E
O algoritmo deve mostrar na tela as notas, a média, o conceito correspondente e a mensagem “APROVADO” se o conceito for A, B ou C ou “REPROVADO” se o conceito for D ou E.

In [ ]:
#define variaveis

nota1 = float(input('Digite a primeira nota: '))
nota2 = float(input('Digite a segunda nota: '))

notas = [nota1, nota2]

#calculo da media usando o len para contar os itens
media = sum(notas)/len(notas)

if media > 9.0:
  aproveitamento = 'A'
elif media >7.5 and 9.0:
  aproveitamento = 'B'
elif media >6.0 and 7.5:
  aproveitamento = 'C'
elif media >4.0 and 6.0:
  aproveitamento = 'D'
else:
  aproveitamento = 'E'

if aproveitamento == 'A':
  status = 'Aprovado'
elif aproveitamento =='B':
  status = 'Aprovado'
elif aproveitamento == 'C':
  status = 'Aprovado'
else:
  status = 'Reprovado'

print('As notas do aluno: ', notas)
print('Média: ', media)
print('Aproveitamento: ', aproveitamento)
print('Status: ', status)

Digite a primeira nota: 6.7
Digite a segunda nota: 7.8
As notas do aluno:  [6.7, 7.8]
Média:  7.25
Aproveitamento:  C
Status:  Aprovado


15. Faça um Programa que peça os 3 lados de um triângulo. O programa deverá informar se os valores podem ser um triângulo. Indique, caso os lados formem um triângulo, se o mesmo é: equilátero, isósceles ou escaleno.
Dicas:
* Três lados formam um triângulo quando a soma de quaisquer dois lados for maior que o terceiro;
* Triângulo Equilátero: três lados iguais;
* Triângulo Isósceles: quaisquer dois lados iguais;
* Triângulo Escaleno: três lados diferentes;

In [ ]:
#define variaveis
lado1 = float(input('Digite o valor para 1° lado: '))
lado2 = float(input('Digite o valor para 2° lado: '))
lado3 = float(input('Digite o valor para 3° lado: '))

#testando se é triângulo
if (lado1 + lado2 < lado3) or (lado1 + lado3 < lado2) or (lado2 + lado3 < lado1):
  print('Não é um triângulo!!')
elif (lado1 == lado2) or (lado1 == lado3):
  print('Triângulo Equilátero!!')
elif (lado1 == lado2) or (lado1 == lado3) or (lado2==lado3):
  print('Triângulo Isósceles!!')
else:
  print('Triângulo Escaleno!!')


Digite o valor para 1° lado: 2
Digite o valor para 2° lado: 2.5
Digite o valor para 3° lado: 2
Triângulo Equilátero!!


16. Faça um programa que calcule as raízes de uma equação do segundo grau, na forma ax2 + bx + c. O programa deverá pedir os valores de a, b e c e fazer as consistências, informando ao usuário nas seguintes situações:


a - Se o usuário informar o valor de A igual a zero, a equação não é do segundo 

grau e o programa não deve fazer pedir os demais valores, sendo encerrado;

b - Se o delta calculado for negativo, a equação não possui raizes reais. Informe ao usuário e encerre o programa;

c - Se o delta calculado for igual a zero a equação possui apenas uma raiz real; informe-a ao usuário;

d - Se o delta for positivo, a equação possui duas raiz reais; informe-as ao usuário;

In [ ]:
import math

print('Equação do 2° grau da forma: ax² + bx + c')

a = int(input('Digite o coeficiente a: '))

if (a==0):
  print('Não é uma equação de segundo grau !')
else:
  b = int(input('Digite o coeficiente b: '))
  c = int(input('Digite o coeficiente c: '))

  delta = b*b - (4*a*c)

  if delta < 0:
    print('A equação não possui raizes reais !')
  elif delta == 0:
    raiz = -b / (2*a)
    print('A equação possui apenas uma raiz real = ', raiz)
  else:
    raiz1 = ( -b + math.sqrt(delta)) / (2*a)
    raiz2 = ( -b - math.sqrt(delta)) / (2*a)
    print('A equação possui duas raizes reais = ', raiz1, 'e', raiz2)

Equação do 2° grau da forma: ax² + bx + c
Digite o coeficiente a: 23
Digite o coeficiente b: 4
Digite o coeficiente c: 0
A equação possui duas raizes reais =  0.0 e -0.17391304347826086


17. Faça um Programa que peça um número correspondente a um determinado ano e em seguida informe se este ano é ou não bissexto.

In [ ]:
ano = int(input('Digite o ano: '))

if (ano%4==0 and ano%100!=0) or ano%400==0:
  print('O Ano ', ano, ' é um ano bissexto!')
else:
  print('O Ano ', ano, ' não é um ano bissexto!')


Digite o ano: 2091
O Ano  2091  não é um ano bissexto!


18. Faça um Programa que peça uma data no formato **dd/mm/aaaa** e determine se a mesma é uma data válida.

In [2]:
dia = int(input('Digite o dia: '))
mes = int(input('Digite o mês: '))
ano = int(input('Digite o ano: '))

validaData = False

if (mes==1 or mes==3 or mes==5 or mes==7 or mes==8 or mes==10 or mes==12):
  if (dia<=31):
    validaData = True

elif (mes==4 or mes==6 or mes==9 or mes==11):
  if (dia<=30):
    validaData= True

elif mes==2:
  if (ano%4==0 and ano%100!=0) or (ano%400==0):
    if (dia<=29):
      validaData = True
    elif (dia<=28):
      validaData = True

if (validaData):
  print('Data digitada é válida!!')
else:  
  print('Data digitada é inválida!! Entre com outra data')

Digite o dia: 25
Digite o mês: 6
Digite o ano: 2025
Data digitada é válida!!


19. Faça um Programa que leia um número inteiro menor que 1000 e imprima a quantidade de centenas, dezenas e unidades do mesmo.
Observando os termos no plural a colocação do "e", da vírgula entre outros. Exemplo:
 * 326 = 3 centenas, 2 dezenas e 6 unidades
 * 12 = 1 dezena e 2 unidades Testar com: 326, 300, 100, 320, 310,305, 301, 101, 311, 111, 25, 20, 10, 21, 11, 1, 7 e 16

In [5]:
num = input ('Digite um número (Menor que 1.000)')
numStr = str (num)
tamanhoNum = len(numStr)

if (tamanhoNum ==3):
  centena = numStr[0:1]
  dezena = numStr[1:2]
  unidade = numStr[2:3]
  print('O número', num, ' digitado representa:', centena,' centenas - ', dezena, ' dezenas - ', unidade, ' unidades!')

if (tamanhoNum ==2):
  centena = numStr[0:1]
  dezena = numStr[1:2]
  print('O número', num, ' digitado representa:', centena,' centenas - ', dezena, ' dezenas!')

if (tamanhoNum ==1):
  centena = numStr[0:1]
  print('O número', num, ' digitado representa:', centena,' centenas!')


Digite um número (Menor que 1.000)456
O número 456  digitado representa: 4  centenas -  5  dezenas -  6  unidades!


20. Faça um Programa para leitura de três notas parciais de um aluno. O programa deve calcular a média alcançada por aluno e presentar:
* a - A mensagem "Aprovado", se a média for maior ou igual a 7, com a respectiva média alcançada;
* b - A mensagem "Reprovado", se a média for menor do que 7, com a respectiva média alcançada;
* c - A mensagem "Aprovado com Distinção", se a média for igual a 10.

In [17]:
nota1 = float(input('Digite a primeira nota do aluno: '))
nota2 = float(input('Digite a segunda nota do aluno: '))
nota3 = float(input('Digite a terceira nota do aluno: '))

notas = [nota1, nota2, nota3]

media = sum(notas)/len(notas)
 
if media == 10 or media == 10.0:
  print('Aluno Aprovado com Distinção média:', round(media,1))
elif media >= 7:
  print('Aluno Aprovado! média: ', round(media,1))
else:
  print('Aluno reprovado! média: ', round(media,1))


Digite a primeira nota do aluno: 10
Digite a segunda nota do aluno: 10
Digite a terceira nota do aluno: 10
Aluno Aprovado com Distinção média: 10.0


21. Faça um Programa para um caixa eletrônico. O programa deverá perguntar ao usuário a valor do saque e depois informar quantas notas de cada valor serão fornecidas. As notas disponíveis serão as de 1, 5, 10, 50 e 100 reais. O valor mínimo é de 10 reais e o máximo de 600 reais. O programa não deve se preocupar com a quantidade de notas existentes na máquina.
* a - Exemplo 1: Para sacar a quantia de 256 reais, o programa fornece duas notas de 100, uma nota de 50, uma nota de 5 e uma nota de 1;
* b - Exemplo 2: Para sacar a quantia de 399 reais, o programa fornece três notas de 100, uma nota de 50, quatro notas de 10, uma nota de 5 e quatro notas de 1.

In [ ]:
#defindo variaveis

saq = int (input('Entre com o valor para saque: '))



if (saq<10):
  print('Valor minimo para saque é 10 - Entre com outro valor')
elif (saq>600):
  print('Valor máximo para saque é de 600 - Entre com outro valor')
else:
  
  num = saq
  cem = int (num/100)
  num = num - (cem*100)

  cinquenta = int (num/50)
  num = num - (cinquenta*50)

  dez = int (num/10)
  num = num - (dez*10)

  cinco = int (num/5)
  num = num - (cinco*5)

  um = num

  print('Valor do saque: ', saq)
  print('Notas 100 = ', cem)
  print('Notas 50 = ', cinquenta)
  print('Notas 10 = ', dez)
  print('Notas 5 = ', cinco)
  print('Notas 1 = ', um)


22. Faça um Programa que peça um número inteiro e determine se ele é par ou impar. Dica: utilize o operador módulo (resto da divisão).

In [28]:
num = int(input('Digite um número: '))
resto = num % 2

if(resto ==0):
    print('O número ', num, ' é par!')
else:
    print('O número ', num, ' é impar!')

Digite um número: 6
O número  6  é par!


23. Faça um Programa que peça um número e informe se o número é inteiro ou decimal. Dica: utilize uma função de arredondamento.

In [26]:
num = float(input('Digite um número: '))

if (num == round(num)):
    print('O número ', num, 'é inteiro')
else:
    print('O número ', num, 'é decimal')
    

Digite um número: 5.65
O número  5.65 é decimal


24. Faça um Programa que leia 2 números e em seguida pergunte ao usuário qual operação ele deseja realizar. O resultado da operação deve ser acompanhado de uma frase que diga se o número é:
* a - par ou ímpar;
* b - positivo ou negativo;
* c - inteiro ou decimal.

25. Faça um programa que faça 5 perguntas para uma pessoa sobre um crime. As perguntas são:
* a - "Telefonou para a vítima?"
* b - "Esteve no local do crime?"
* c - "Mora perto da vítima?"
* d - "Devia para a vítima?"
* e - "Já trabalhou com a vítima?" O programa deve no final emitir uma classificação sobre a participação da pessoa no crime. Se a pessoa responder positivamente a 2 questões ela deve ser classificada como "Suspeita", entre 3 e 4 como "Cúmplice" e 5 como "Assassino". Caso contrário, ele será classificado como "Inocente".

26. Um posto está vendendo combustíveis com a seguinte tabela de descontos:
* a - Álcool:
* b - até 20 litros, desconto de 3% por litro
* c - acima de 20 litros, desconto de 5% por litro
* d - Gasolina:
* e - até 20 litros, desconto de 4% por litro
* f - acima de 20 litros, desconto de 6% por litro Escreva um algoritmo que leia o número de litros vendidos, o tipo de combustível (codificado da seguinte forma: A-álcool, G-gasolina), calcule e imprima o valor a ser pago pelo cliente sabendo-se que o preço do litro da gasolina é R 2.50 o preço do litro do álcool é R$ 1.90.

27. Uma fruteira está vendendo frutas com a seguinte tabela de preços:
                      Até 5 Kg           Acima de 5 Kg
          Morango         R$ 2,50 por Kg          R$ 2,20 por Kg
          Maçã            R$ 1,80 por Kg          R$ 1,50 por Kg

Se o cliente comprar mais de 8 Kg em frutas ou o valor total da compra ultrapassar R$ 25,00, receberá ainda um desconto de 10% sobre este total. Escreva um algoritmo para ler a quantidade (em Kg) de morangos e a quantidade (em Kg) de maças adquiridas e escreva o valor a ser pago pelo cliente.

28. O Hipermercado Tabajara está com uma promoção de carnes que é imperdível. Confira:
                      Até 5 Kg           Acima de 5 Kg
        File Duplo      R$ 4,90 por Kg          R$ 5,80 por Kg
        Alcatra         R$ 5,90 por Kg          R$ 6,80 por Kg
        Picanha         R$ 6,90 por Kg          R$ 7,80 por Kg


Para atender a todos os clientes, cada cliente poderá levar apenas um dos tipos de carne da promoção, porém não há limites para a quantidade de carne por cliente. Se compra for feita no cartão Tabajara o cliente receberá ainda um desconto de 5% sobre o total da compra. Escreva um programa que peça o tipo e a quantidade de carne comprada pelo usuário e gere um cupom fiscal, contendo as informações da compra: tipo e quantidade de carne, preço total, tipo de pagamento, valor do desconto e valor a pagar.